.gitignore

```
@@ -0,0 +1 @@
+/node_modules
```

If you are using a Mac you should ignore `.DS_Store` file too!

```suggestion
/node_modules
.DS_Store
```